Bottom: a9579c1a909058c37801f47774809bacbbf9ff51
Top:    d4a4d22f610d6665a1923f3f3acdaf7ce5ba0af1
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 12:19:58 -0500

Remove container conf directory

Roles create it themselves


---
diff --git a/roles/docker-common/tasks/main.yml b/roles/docker-common/tasks/main.yml
index 7f9e8f5..e955efa 100644
--- a/roles/docker-common/tasks/main.yml
+++ b/roles/docker-common/tasks/main.yml
@@ -9,13 +9,6 @@
 - import_tasks: sshd_settings.yml
 - import_tasks: mariadb_settings.yml
 
-- name: base docker_container.conf dir
-  file:
-    state: directory
-    path: '{{ docker_restore_config_base_dir }}'
-    owner: root
-    group: root
-    mode: 'u=rwx,g=rx,o=rx'
 
 
 - name: Check if baculavirt selinux module exists
